thunder/executors/torchex.py

```diff
@@ -532,6 +545,30 @@ def _squeeze_transform(a: TensorLike, /, dim: None | int | Sequence[int] = None)
     return squeeze(a, dim)
 
 
+def _empty_transform(
+    shape: tuple[int, ...],
```

Comment:
nit: `Sequence[int]` here -- a list would be accepted, for example

Reply:
Thought prims and empty had the same type for shape. Thank you for catching! Let me focus on to the details next time.